test/parallel/test-http2-getpackedsettings.js

```diff
@@ -157,6 +161,7 @@ http2.getPackedSettings({ enablePush: false });
   assert.strictEqual(settings.maxHeaderSize, 100);
   assert.strictEqual(settings.enablePush, true);
   assert.strictEqual(settings.enableConnectProtocol, false);
+  assert.deepStrictEqual(settings.customSettings, { '9999': 301 });
```

Comment:
There should be a test here verifying multiple custom settings can be set, settings with invalid keys/values are rejected, and too many custom settings throw the appropriate error.

Reply:
Test were added in the changed commits